libs/core/langchain_core/prompts/chat.py

```diff
@@ -406,22 +406,27 @@ def get_lc_namespace(cls) -> List[str]:
     def from_template(
         cls: Type[_StringImageMessagePromptTemplateT],
         template: Union[str, List[Union[str, _TextTemplateParam, _ImageTemplateParam]]],
+        *,
```

Comment:
> This is a breaking change since template_format was a positional argument.
> 
> Let's move the `*` after the `template_format` variable, so it doesn't break any existing code.

thanks for the explanation. Done.